Sources/HipMobileUI/ViewModels/Pages/UserRatingPageViewModel.cs

```diff
@@ -93,59 +92,51 @@ public UserRatingPageViewModel(Exhibit exhibit)
 
         private async void SetUserRatingUi()
         {
-            UserRating userRating = await IoCManager.Resolve<IUserRatingManager>().GetUserRatingAsync(exhibit.IdForRestApi);
+            var userRating = await IoCManager.Resolve<IUserRatingManager>().GetUserRatingAsync(exhibit.IdForRestApi);
```

Comment:
This is theoretically not possible because you can't go to the user rating page without a network connection and you can only refresh the page by clicking on "Bewerten" which also checks your network connection before the site is going to be refreshed. Nevertheless, I have added a check for this, just in case.